pages/linux/debootstrap.md

```diff
@@ -0,0 +1,16 @@
+# debootstrap
+
+> Create a basic Debian system.
+> More information: <https://wiki.debian.org/Debootstrap>.
+
+- Create a Debian buster system inside {{buster-chroot}} directory:
```

Comment:
I prefer this highlight mode:
```suggestion
- Create a Debian buster system inside `buster-chroot` directory:
```

Reply:
@navarroaxel 
It seems for me backtick takes no effect here with [python client](https://github.com/tldr-pages/tldr-python-client):
![image](https://user-images.githubusercontent.com/14995027/99879310-f6c22080-2c2d-11eb-9cbf-d6af13b64280.png)